glib/src/lib.rs

```diff
@@ -199,7 +199,7 @@ pub use self::bridged_logging::{rust_log_handler, GlibLogger, GlibLoggerDomain,
 pub mod subclass;
 
 mod main_context_futures;
-pub use main_context_futures::{JoinError, JoinHandle};
+pub use main_context_futures::{spawn, spawn_with_priority, JoinError, JoinHandle};
```

Comment:
The naming of the functions is a bit suboptimal as there's also this set of functions: https://gtk-rs.org/gtk-rs-core/git/docs/glib/fn.spawn_async.html

Any suggestions?

Reply:
Or directly make these two functions methods of `MainContext` and add doc aliases on them.

Reply:
There are already methods with that name on `MainContext`. The goal here is to have a function that gets the correct main context instance and then calls the method on that.

Reply:
`spawn_on_thread_default_main_context()` could work of course, but that's almost as long as the implementation of the function :)

Reply:
Maybe could move this into a `glib::futures` module, and also potentially move some other things in there. Not sure.